virttest/qemu_storage.py

```diff
@@ -1534,10 +1534,12 @@ def check(self, params, root_dir, force_share=False, output=None):
         image_filename = self.image_filename
         LOG.debug("Checking image file %s", image_filename)
         force_share &= self.cap_force_share
+        check_repair = self.params.get("image_check_repair")
```

Comment:
Hi @zhencliu @hellohellenmao 
I suggest adding a new argument "check_repair=None" for the definition.
`def check(self, params, root_dir, force_share=False, output=None, check_repair=None):`
From my understanding, the attribute `check_repair` doesn't belong to the image namespace, not like `image_size`, `image_format`, `image_filename`, and so on.
The `check_repair` just focuses on the operation checking of the image, it's not a common attribute. So I think shall we add a new argument for this options?
Thanks.

Reply:
@zhencliu What do you think?

Reply:
hi @YongxueHong , agreed. I considered to add a new arg, I thought that once we want to repair an image in test cases, we have to update py files to give a new arg. But I never considered that if check_repair is a proper attribute of an image, yes, it's not an attribute of an image.  hi @hellohellenmao , what's your opinion?

Reply:
Agree, and okay from my side. I will update it soon. Thanks all.